model-engine/model_engine_server/inference/forwarding/http_forwarder.py

```diff
@@ -36,14 +39,18 @@ def get_config():
     )
 
 
-def get_forwarder_loader():
+def get_forwarder_loader(destination_path: Optional[str] = None):
     config = get_config()
+    if destination_path:
```

Comment:
wait does this do anything? since we look at `config['sync']` instead of `config['predict_route']`

Reply:
ah typo

Reply:
also get rid of the ["sync"] in `config["sync"]["predict_route"] = destination_path`?